independent-publisher-connectors/rSpaceX/readme.md

```diff
@@ -0,0 +1,97 @@
+# r/SpaceX
+r/SpaceX is an open Source REST API for SpaceX launch, rocket, core, capsule, Starlink, launchpad, and landing pad data. r/SpaceX is not affiliated, associated, authorized, endorsed by, or in any way officially connected with Space Exploration Technologies Corp (SpaceX), or any of its subsidiaries or its affiliates. The names SpaceX as well as related names, marks, emblems and images are registered trademarks of their respective owners.
+
+## Publisher: Troy Taylor, Hitachi Solutions

```

Comment:
@troystaylor This section is not required in the readme.md

Reply:
This section may not be required, but it the disclaimer is necessary as this an unofficial API for SpaceX.